packages/core/types/src/modules/documents/params/index.ts

```diff
@@ -42,7 +42,7 @@ export type Pick<
     // Publication State
     [HasMember<TKind, 'status'>, PublicationState.Param],
     // Locale
-    [HasMember<TKind, 'locale'>, { locale?: string }],
```

Comment:
I might add the type locale in this file, rather than in attributes, as attributes hold each attribute type you can have in a schema